client/reader/stream/index.jsx

```diff
@@ -470,9 +409,13 @@ class ReaderStream extends React.Component {
 			);
 			showingStream = true;
 		}
+		const indexOfColon = streamKey.indexOf( ':' );
```

Comment:
maybe make a helper that returns a pair of stream type and stream id?

Reply:
i actually have a bunch of unpushed changes right now that extract this function into one called `getStreamType`